docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,60 @@
+# Why TestCafe?
+
+TestCafe excels where other end-to-end testing solutions fall short.
+
+## One NPM package
+
+TestCafe doesn't require the use of extra tools and frameworks. All you need to get up and running is a recent version of Node.js, a browser — any browser, and the TestCafe NPM package.
+
+## As simple and extendable as JavaScript itself
+
+TestCafe tests are full-fledged Node.js apps. They are compatible with your favourite JavaScript libraries and preprocessors.
+
+## Complex tests made easy
+
+TestCafe’s intuitive and powerful toolkit makes testing easy, whether you’re working with a landing page or a complex web app.
+
+### Mock Requests
+Emulate HTTP responses to feed your app sample data, troubleshoot connectivity errors and cheat downtime.
+
+### Client-side injections
+Execute custom client-side code to parse the web page, examine its state, or even add extra dependencies.
+
+### Frames and windows
+Make use of multiple browser windows to test out complex user interactions. Switch to and from iframes with ease and stability.
+
+### Concurrent tests
+Run your tests in multiple browsers at once to quickly uncover browser-specific errors.
+
+## Automate what’s possible
+
+### Automated waiting
+Manual timeouts are a thing of the past. Asynchronous from the ground-up, TestCafe knows when to wait and what to wait for before the test can continue.
```

Comment:
Can we say that manual timeouts are a thing of the past and... implement [t.wait](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/wait.html) at the same time?

The message is right, but maybe we can say something less unconditional, like *Manual timeouts make your tests unstable.*

Reply:
Can we simply remove this sentence? Or do we need to justify the automated waiting feature?